cmd/bucket-replication.go

```diff
@@ -200,6 +205,11 @@ func replicateObject(ctx context.Context, bucket, object, versionID string, obje
 		gr.Close()
 		return
 	}
+
+	// if heal encounters a pending replication status, either replication
+	// has failed due to server shutdown or crawler and PutObject replication are in contention.
```

Comment:
Seems redundant with the comment below?

Reply:
yes its kept there to get more eyes, but yeah we can remove it later.